python/api-examples-source/charts.altair_selections.py

```diff
@@ -0,0 +1,26 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+import altair as alt
+
+
+@st.cache_data
+def load_data():
+    return pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+
+
+df = load_data()
+
+chart = (
+    alt.Chart(df)
+    .mark_circle()
+    .encode(x="a", y="b", size="c", color="c", tooltip=["a", "b", "c"])
+    .add_params(
```

Comment:
nit: `altair==4.2.0` pinned in requirements does not support `add_params`. Need to bump altair to v5+

Reply:
Upgraded altair and used a whl file from develop since streamlit-nightly will cause altair to downgrade on Community Cloud.